### ISSUE TITLE
[tempo-distributed] Include Grafana Dashboards for monitoring

### ISSUE THREAD

**Issue by braunsonm** [grafana/helm-charts#2245]
Unlike the Loki charts, the Tempo distributed does not include the Grafana dashboards required for effectively monitoring Tempo. it would be nice if these were deployed as part of the chart.

**Comment by SennaSemakula** [comment]
Agreed. Raising this as well as I would like https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/dashboards to be included as part of https://github.com/grafana/helm-charts/tree/main/charts/tempo-distributed. For mimir and loki charts we do this so can someone look at picking this up?
